spring-integration-core/src/test/java/org/springframework/integration/util/SimplePoolTests.java

```diff
@@ -16,18 +16,18 @@
 
 package org.springframework.integration.util;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.fail;
-
+import java.util.ArrayList;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 import java.util.concurrent.Semaphore;
 import java.util.concurrent.atomic.AtomicBoolean;
-
 import org.junit.Test;
-
 import org.springframework.integration.test.util.TestUtils;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
```

Comment:
Please, do not rearrange imports order.
Consider to run `gradlew check --parallel` task to be sure that your changes follows our Checkstyle rules.
Also that will say you if your fix is valid against all the existing tests in the project.

Reply:
sorry, I tried to keep imports untouched. I will revert this